router.js

```diff
@@ -0,0 +1,85 @@
+const router = require("express").Router();
+const dbHelper = require("./data/dbHelper");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const env = require("dotenv");
+env.config();
+
+router.get("/", (req, res) => {
+  res.status(200).json({ message: "The router at /api is working" });
+});
+
+router.get("/users", (req, res) => {
+  dbHelper
+    .getUsers()
+    .then(users => {
+      res.status(200).json(users);
+    })
+    .catch(err => {
+      res.status(500).json(err.message);
+    });
+});
+
+router.post("/register", (req, res) => {
+  const username = req.headers.username;
+  const password = req.headers.password;
+
+  const hash = bcrypt.hashSync(password, 10);
+
+  dbHelper
+    .register({ username, password: hash })
+    .then(id => {
+      res
+        .status(200)
+        .json({ message: `${username} created with id # : ${id}` });
```

Comment:
inside this then, you could call your login helper method and attach a token here so the user doesnt need to then go login to receive the token